packages/nextjs/tailwind.config.js

```diff
@@ -30,6 +30,19 @@ module.exports = {
           ".tooltip": {
             "--tooltip-tail": "6px",
           },
+
+          [`input:-webkit-autofill,
```

Comment:
small nitpick(maybe we can ignore it) : 

main branch : 

https://user-images.githubusercontent.com/80153681/218668808-5ca6c102-e6b9-4cc1-9797-4c0e7223d321.mov

This branch : 

https://user-images.githubusercontent.com/80153681/218668840-77cce80e-9001-44d7-bb1c-72afd4e8b624.mov

I see the only differece is while selecting atuocomplete it highlights input with different color, once the autocomplete is selected both branch looks same(just didn't wanted to ovveride defaults)

please don't remove this yet. Let's hear from @carletex first, I don't think its much of an issue even if we keep it